### PR TITLE
Use io.WriteString

### PR DESCRIPTION
**Pull request by vbauerster**
There is better way to write a string to io.Writer

### CODE REVIEW

**Comment by vbauerster** [comment]
You're right, but before conversion, it queries for `WriteString(s string) (n int, err error)` behavioural contract, and if type assertion succeeds `WriteString` method is called. 

Behavioural contract is just assumption that, `WriteString` method would allow strings to be efficiently written, avoiding the need to allocate a temporary copy.

In practice, a number of important types that satisfy `io.Writer` also have a `WriteString` method, including `*bytes.Buffer`, `*os.File` and `*bufio.Writer`.  Therefore using `io.WriteString` may improve performance at most, and at least string will be written any way.

**Comment by bahlo** [comment]
That‘s right, can you add a short sentence about this to the reason in the README? Also please merge (or rebase) master in your branch, there are conflicts. 👍🏻

File: README.md [hunk 1]
```diff
@@ -102,11 +103,11 @@ soon become part of the toolchain.
 
 ### Use Semantic Versioning
 Since `dep` can handle versions, tag your packages using
-[Semantic Versioning](http://semver.org).  
+[Semantic Versioning](http://semver.org).
```
[the remark this answers is not in the document]
Comment (vbauerster) [reply README.md]: Done, double space reverted.

File: README.md [hunk 2]
```diff
@@ -768,6 +769,28 @@ To enable single-binary deployments, use tools to add templates and other static
 assets to your binary
 (e.g. [github.com/jteeuwen/go-bindata](https://github.com/jteeuwen/go-bindata)).
 
+## Use io.WriteString
```
Comment (bahlo) [inline README.md]: Sorry, another thing: Can you wrap it in backticks?

Comment (vbauerster) [reply README.md]: Yes, sure. Thank you for noting this.

Comment (vbauerster) [reply README.md]: done

**Pull request merged**